src/components/Main.js

```diff
@@ -89,7 +88,7 @@ export default class Main extends React.Component {
   }
 }
 
-function injectCSS (url) {
+function injectCSS(url) {
```

Comment:
if we're going to ever inject another JS file into the page, yes. [the `injectCSS` calls at the top](https://github.com/aframevr/aframe-inspector/pull/272/files#diff-ea337f34aa79e31b41de2123664a0de4R16) just struck me as odd. it's fine to keep for now.